tests/admin/test_webauthn.py

```diff
@@ -0,0 +1,92 @@
+from .. import util
+import duo_client.admin
+from .base import TestAdmin
+
+
+class TestWebauthn(TestAdmin):
+    def test_get_webauthncredentials_with_params(self):
+        """ Test to get webauthn credentials with params.
+        """
+        response = list(self.client_list.get_webauthncredentials(limit=8))[0]
+        uri, args = response['uri'].split('?')
+
+        self.assertEqual(response['method'], 'GET')
+        self.assertEqual(uri, '/admin/v1/webauthn-credentials')
+        self.assertEqual(
+            util.params_to_dict(args),
+            {
+                'account_id':[self.client_list.account_id],
+                'limit': ['8'],
+                'offset': ['0'],
+            }
+        )
+
+    def test_get_webauthncredentials_iterator(self):
+        response = self.client_list.get_webauthncredentials_iterator()
+        response = next(response)
+        uri, args = response['uri'].split('?')
+
+        self.assertEqual(response['method'], 'GET')
+        self.assertEqual(uri, '/admin/v1/webauthn-credentials')
+        self.assertEqual(
+            util.params_to_dict(args),
+            {
+                'account_id': [self.client_list.account_id],
+                'limit': ['100'],
+                'offset': ['0']
+            }
+        )
+
+    def test_get_webauthncredentials_without_params(self):
+        """ Test to get webauthn credentials without params.
+        """
+        response = list(self.client_list.get_webauthncredentials())[0]
+        uri, args = response['uri'].split('?')
+
+        self.assertEqual(response['method'], 'GET')
+        self.assertEqual(uri, '/admin/v1/webauthn-credentials')
+        self.assertEqual(
+            util.params_to_dict(args),
+            {
+                'account_id': [self.client_list.account_id],
+                'limit': ['100'],
+                'offset': ['0'],
+             }
+        )
+
+    def test_get_webauthncredentials_with_offset(self):
+        response = list(self.client_list.get_webauthncredentials(limit=2, offset=3))[0]
+        uri, args = response['uri'].split('?')
+
+        self.assertEqual(response['method'], 'GET')
+        self.assertEqual(uri, '/admin/v1/webauthn-credentials')
```

Comment:
This will also need to be updated to `/admin/v1/webauthncredentials` based on the above comment.